Bookmark/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+import datetime
+class Bookmark(models.Model):
+
+    date_created = models.DateField("Created time")
```

Comment:
I’m not sure if  Created Time is the correct format here. Have you tested this code by adding bookmarks?